crowbar_framework/config/initializers/delayed_job_config.rb

```diff
@@ -14,6 +14,20 @@
 # limitations under the License.
 #
 
+# Enable proper traceback logging of delayed jobs
+module Delayed
+  class Worker
+    def handle_failed_job_with_logging(job, error)
+      handle_failed_job_without_logging(job, error)
+      Rails.logger.error("delayed_job #{job.name} with ID #{job.id} failed. "\
+        "See the delayed_job log for more details.")
```

Comment:
The log file is called `background_jobs.log` (I do not care if you rename it or change the text here, btw)